### ISSUE TITLE
v14.0.1 break the folder upload function

### ISSUE THREAD

**Issue by yangfan0356** [georgipeltekov/ngx-file-drop#268]
Tested both v14.0.1 and v14.0.0.  
V14.0.0 works fine, but V14.0.1 does not work on "folder" upload anymore. Can be tested using the demo example.

**Comment by babette-landmesser** [comment]
For me, the version 14.0.0 doesn't work either. 13.0.0 worked like a charm but since upgrading to 14, the folder upload is broken.
When uploading a folder, the `onFileDrop` event returns this array:
```
[
    {
        "relativePath": "folder1",
        "fileEntry": {
            "name": "folder1",
            "isDirectory": false,
            "isFile": true
        }
    }
]
```

It's a folder on my system and it has files inside. As you can see, even the 'isDirectory' value is returned false. 

When switching back to version 13, the exact same function returns the following array:
```
[
    {
        "relativePath": "folder1/image1.png",
        "fileEntry": {}
    },
    {
        "relativePath": "folder1/image2.png",
        "fileEntry": {}
    }
]
```

I hope this additional info helps to find the solution :)

**Comment by georgipeltekov** [comment]
The regression should be fixed in the latest 14.0.2, pushed just now
